contracts/modules/TransferManager/MATM/ManualApprovalTransferManager.sol

```diff
@@ -230,6 +231,7 @@ contract ManualApprovalTransferManager is ManualApprovalTransferManagerStorage,
                     allowance = allowance - _changeInAllowance;
                 }
             }
+            approval.approvedAllowance = allowance;
```

Comment:
Sorry to re-iterate @remon-nashid but `initialAllowance` will be the better name for this

Reply:
And one more thing, `approval.approvedAllowance` value should be changed as per the increase or decrease i.e
```
       if increase 
       approvedAllowance =  approvedAllowance + changeInAllowance
       if decrease
       approvedAllowance =  approvedAllowance - changeInAllowance
```

Reply:
Done